packages/graphql/src/schemaTypes/objectTypes/gql-Mutation.ts

```diff
@@ -684,6 +684,25 @@ export const mutation = mutationType({
       },
     })
 
+    t.field('insertCohort', {
+      type: 'Boolean',
+      description: '',
+      args: {
+        name: nonNull(stringArg()),
+        cohort: nonNull(stringArg()),
+      },
+      resolve: async (source, args, ctx) => {
+        const cohort = {
+          name: args.name,
+          cohort: args.cohort,
+        }
+
+        await ctx.actions.cohorts.insertCohort(cohort)
+
+        return true
```

Comment:
Is the expectation to just throw in the event of an error? Or should this catch and resolve `false`?